gateways/clients/wallet_service_db_client.py

```diff
@@ -48,11 +48,16 @@
        token.name AS name,
        token.symbol AS symbol,
        COUNT(*) OVER() as total
-FROM token RIGHT OUTER JOIN address_balance ON token.id = address_balance.token_id
+FROM token INNER JOIN address_balance ON token.id = address_balance.token_id
```

Comment:
This was changed to an `INNER JOIN` because it will not include HTR.
Since we already search HTR outside of this query.

Reply:
After https://github.com/HathorNetwork/hathor-wallet-service/pull/273 is merged this will also include HTR.

Should we explicitly exclude HTR? As in add a `WHERE token.id != '00'`

Reply:
I think that should work